Gems/Atom/RPI/Code/Include/Atom/RPI.Public/RenderPipeline.h

```diff
@@ -201,6 +214,19 @@ namespace AZ
             //! use RPI::PassSystemInterface::Get()->ForEachPass() function instead.
             Ptr<Pass> FindFirstPass(const AZ::Name& passName);
 
+            //! Finding, adding and removing pipeline attachments (attachments that can be referenced directly by any pass in the pipeline)
+            const Ptr<PassAttachment> GetPipelineAttachment(const Name& name) const;
```

Comment:
Again nit, but 'GetGlobalAttachments' is much more clear than 'GetPipelineAttachment' that is open for interpretation.

Reply:
I prefer pipeline attachment since I think global attachment implies the attachment is accessible across different render pipelines.

Reply:
In this case how about 'GetPipelineGlobalAttachments'?  it implies that it is pipeline specific but Global